### PR TITLE
Fixes #19

### LINKED ISSUE #19: Allow root user to modify constants defined during initial setup later

### PR DESCRIPTION
**Pull request by vpn1997**
Added Root-settings button that directs the user to root settings page.

image

### CODE REVIEW

**Comment by AvaisP** [comment]
The root settings are only meant for the root user. It shouldn't be available for every user

**Comment by vpn1997** [comment]
But "initialsetup" route is available to all the users .And can you clarify what do you mean by root user?

**Comment by vpn1997** [comment]
Hey @AvaisP  I have done some changes .Now only root user will be able to see Root-Settings button . :smile:

**Comment by AvaisP** [comment]
@vpn1997 Instead of just not displaying the Root Settings menu item, we want the initialsetup url to be inaccessible to anyone who isn't the root user. Note: If root user isn't set, then the url must be accessible to anyone

File: Origami/src/components/App.js [hunk 1]
```diff
@@ -24,8 +24,10 @@ class App extends React.Component {
       login: false,
       displayLogin: "",
       showTitle: true,
-      isFrame: window.location.pathname.split("/")[1] == "frame"
+      isFrame: window.location.pathname.split("/")[1] == "frame",
+      isRoot:false
```
Comment (AvaisP) [inline Origami/src/components/App.js]: Instead of using states here, don't allow anyone else apart from the root user to access the URL

Comment (vpn1997) [reply Origami/src/components/App.js]: ohk ill do the necessary changes.

**Comment by vpn1997** [comment]
@AvaisP @uttu357  I have done some changes. Now only root user can  access "/initialsetup" route. :smile:

**Comment by AvaisP** [comment]
The root user should neither be able to view the rootsettings through the url nor should the option appear in the Nav Bar

**Comment by vpn1997** [comment]
@AvaisP done

**Comment by vpn1997** [comment]
@AvaisP Done.

**Pull request merged**